WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -406,6 +407,29 @@ public static void viewStatsInNewStack(Context context, SiteModel site) {
         taskStackBuilder.startActivities();
     }
 
+    public static void viewStatsInNewStack(Context context, SiteModel site, StatsTimeframe statsTimeframe) {
+        if (site == null) {
+            AppLog.e(T.STATS, "SiteModel is null when opening the stats from the deeplink.");
+            AnalyticsTracker.track(
+                    STATS_ACCESS_ERROR,
+                    ActivityLauncher.class.getName(),
+                    "NullPointerException",
+                    "Failed to open Stats from the deeplink because of the null SiteModel"
+                                  );
+            ToastUtils.showToast(context, R.string.stats_cannot_be_started, ToastUtils.Duration.SHORT);
+            return;
+        }
+        TaskStackBuilder taskStackBuilder = TaskStackBuilder.create(context);
+
+        Intent mainActivityIntent = getMainActivityInNewStack(context);
+
+        Intent statsIntent = StatsActivity.buildIntent(context, site, statsTimeframe);
+
+        taskStackBuilder.addNextIntent(mainActivityIntent);
+        taskStackBuilder.addNextIntent(statsIntent);
+        taskStackBuilder.startActivities();
+    }
```

Comment:
This is not critical at all, but I wish there was a way for us to skip the transition animation when using `TaskStackBuilder`, as it looks kinda weird right now:

https://user-images.githubusercontent.com/830056/114952494-c2454000-9e2c-11eb-9be7-81ee9f826089.mp4

I tried adding a `Intent.FLAG_ACTIVITY_NO_ANIMATION` to the `mainActivityIntent` but it didn't work. Just wondering if you know a way to achieve this?